Github/service.js

```diff
@@ -167,3 +168,82 @@ module.exports.getAllCommitsTogether = branches => {
       });
   });
 };
+
+const getIssues = repo => {
+  return new Promise((resolve, reject) => {
+    request.get(
+      {
+        url: `https://api.github.com/repos/Seneca-CDOT/${repo}/issues?labels=help+wanted&access_token=${key}`,
+        headers: { 'User-Agent': 'request' },
+      },
+      (err, res, data) => {
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error('Unable to get repos.'));
+        } else {
+          const filteredIssues = [];
+          JSON.parse(data).forEach(issue => {
+            const assigs = [];
+            if (issue.assignees.length !== 0) {
+              issue.assignees.forEach(assignee => {
+                assigs.push({
+                  name: assignee.login,
+                  avatar: assignee.avatar_url,
+                });
+              });
+            }
+            filteredIssues.push({
+              ra: issue.user.login,
+              repository: issue.repository_url.slice(
+                issue.repository_url.lastIndexOf('/') + 1,
+                issue.repository_url.length,
+              ),
+              number: issue.number,
+              title: issue.title,
+              description: issue.body,
+              label: 'Help Wanted',
+              state: issue.state,
+              assignees: assigs,
+              milestone: issue.milestone,
+              created: new Date(issue.created_at).toLocaleString('en-US', {
+                timeZone: 'America/Toronto',
+              }),
+              updated: new Date(issue.updated_at).toLocaleString('en-US', {
+                timeZone: 'America/Toronto',
+              }),
+            });
+          });
+          resolve(filteredIssues);
+        }
+      },
+    );
+  });
+};
+
+module.exports.getFilteredIssues = repos => {
+  const issues = [];
+  return new Promise((resolve, reject) => {
+    const promises = [];
+    repos.forEach(repo => {
+      promises.push(getIssues(repo.name));
+    });
+    Promise.all(promises)
+      .then(arraysOfIssues => {
+        arraysOfIssues.forEach(arrayOfIssues => {
```

Comment:
I'd recommend [flattening](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) the arrays so you don't have to deal with a nested forEach.